projects/PM_hash_navigation/main.cc

```diff
@@ -48,7 +50,7 @@ angleNormalise(int angle)
 }
 
 int
-main(int argc, char **argv)
+bobMain(int argc, char **argv)
```

Comment:
We've started using ``bobMain`` functions instead of ``main`` ones, because that way you can make sure the robot stops moving if you get an error.